### PR TITLE
always_run is deprecated. Use check_mode = no instead..

### PR DESCRIPTION
**Pull request by Brzhk**
[DEPRECATION WARNING]: always_run is deprecated. Use check_mode = no instead..
This feature will be removed in version 2.4. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -107,7 +107,7 @@
     LC_ALL: 'C'
   shell: dpkg-divert --list '{{ php__etc_base + "/fpm/*" }}' | awk '{print $NF}'
   register: php__register_diversions
-  always_run: True
+  check_mode: no
```
Comment (drybjed) [inline tasks/main.yml]: This PR was not updated, apparently.

**Comment by mbergumi** [comment]
@drybjed : fixed.

**Comment by drybjed** [comment]
@mbergumi Not fixed. :-)

**Comment by mbergumi** [comment]
?!

**Comment by mbergumi** [comment]
ok got it. for some reason this repository had a different branch name for the PR, and therefore my batch-sed ops applied changes on the wrong branch here. it should be OK now, but aint got the time to check really right now

**Comment by drybjed** [comment]
@mbergumi OK, that looks good. Can also explain the relationship between this account and @Brzhk? I'm not sure how you were able to push commits to this PR...

**Comment by mbergumi** [comment]
same guy, different avatar. ^^ long and boring story, but.. work policy.

**Review by drybjed — APPROVED** [review]

**Pull request merged**